### ISSUE TITLE
xWebAdministration: Move example from README.md to examples folder

### ISSUE THREAD

**Issue by johlju** [PowerShell/xWebAdministration#485]
This example https://github.com/PowerShell/xWebAdministration#registering-php should be moved to the Examples folder. Preferably in the root of the Examples folder as an end-to-end example.

*Please note that examples are tested so they can be compiled.*

**Comment by Tuxhedoh** [comment]
I think I'd like to take a crack at this. Is this literally just taking the example and putting it in it's own ps1 file in the examples folder? I'm assuming that you'd still want the PHP instructions on the page, with a link to the example.

**Comment by johlju** [comment]
@Tuxhedoh Yeah, just put in the root of the Examples folder. Make sure the name says something with end-to-end so we can differentiate it from the individual resource examples (that should go in separate folders mentioned in another issue).

For the text, please add a comment-based help block as seen here.
https://github.com/PowerShell/ActiveDirectoryDsc/blob/3e96da395d40c8667702b03a6b8665dc1ee35486/Examples/Resources/ADComputer/1-ADComputer_AddComputerAccount_Config.ps1#L20-L24

**Comment by johlju** [comment]
All text goes in the comment-based help block, and remove it from the README.md entirely.

**Comment by Tuxhedoh** [comment]
What's the proper etiquette when it comes to handling multiple issues in one repo? Make one PR and resolve them all? There seems to be quite a bit of low hanging fruit in this project. I recognize with Hacktoberfest there will probably be quite a few people clamoring for the issues to get their 4 pr's.

**Comment by johlju** [comment]
Just mention in an issue that your are working on it, then you may send in one PR that resolves them all or several PR's that solves one each. Normally I personally send in a PR fixing a bunch of PR's.

I expect during October to get small PR's resolving one issue. That was my meaning with all these `hacktoberfest` issues. I also hope new contributors (to GitHub) feel they want to try send in their first PR too.

**Comment by johlju** [comment]
I have not seen a PR for this issue so I have labeled it help wanted so that anyone can pick it up.

**Comment by DennisL68** [comment]
The resource xWebAdministration has been renamed.
Bug fixes and new functionality will only be added to the renamed resource.
